fabtests/benchmarks/benchmark_shared.c

```diff
@@ -72,7 +81,11 @@ void ft_benchmark_usage(void)
 
 int pingpong(void)
 {
-	int ret, i;
+	int ret, i, inject_size;
+
+	inject_size = option_j_set ?
+			MIN(hints->tx_attr->inject_size, fi->tx_attr->inject_size) :
```

Comment:
We shouldn't need the MIN now.  Always use hints->inject_size.  The fi->inject_size should be >= hints, or something is wrong with getinfo.

Reply:
updated.